src/formatDate.js

```diff
@@ -9,6 +9,41 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+
+  const splitedDate = date.split(oldSeparator);
```

Comment:
The variable name `splitedDate` appears to be a typo. Consider renaming it to `splitDate` to enhance readability.